README.md

```diff
@@ -88,9 +88,16 @@ After logging in, use [localhost admin](https://127.0.0.1:8080/admin) to modify
 flake8 lms
 ```
 ### Run tests
+Running tests inside the docker container:
+```
```

Comment:
Why would you want to run the tests inside the docker? :)

Reply:
Since all dependencies installation already done there.
Do you see any reason why not?

Reply:
You can run it on your own fs, and the development shouldn't happen inside the docker anyway

Reply:
What do you think about this approach?
https://www.docker.com/blog/containerizing-test-tooling-creating-your-dockerfile-and-makefile/

Reply:
I removed this section as part of #179